src/components/views/messages/MLocationBody.tsx

```diff
@@ -97,11 +97,13 @@ export const LocationBodyFallbackContent: React.FC<{ event: MatrixEvent, error:
         (_t('Shared a location: ') + event.getContent()?.body);
 
     return <div className="mx_EventTile_body">
-        <span className={errorType !== LocationShareError.MapStyleUrlNotConfigured ? "mx_EventTile_tileError" : ''}>
+        <span className={errorType !== LocationShareError.MapStyleUrlNotConfigured
+            ? "mx_EventTile_tileError mx_MLocationBody_error"
+            : ''
+        }>
             { message }
         </span>
-        <br />
-        { locationFallback }
+        <div className="mx_MLocationBody_fallback">{ locationFallback }</div>
```

Comment:
To fix this (fallback message)

![fallback](https://user-images.githubusercontent.com/3362943/166210237-ae66e705-c599-48d4-ba5d-8741e3df37cb.png)